src/z3c/schema/baseurl/tests.py

```diff
@@ -46,15 +46,15 @@ def testBadStringType(self):
     def test_newlines(self):
         field = self._Field_Factory(title=u'BaseURL field')
         self.assertRaises(InvalidBaseURL, field.validate,
-            self._convert('host\nfoo'))
+                          self._convert('host\nfoo'))
 
 
 def test_suite():
     return unittest.TestSuite((
         doctest.DocFileSuite(
-                'README.txt',
-                optionflags=doctest.NORMALIZE_WHITESPACE|\
-                            doctest.ELLIPSIS|\
-                            doctest.IGNORE_EXCEPTION_DETAIL),
+            'README.txt',
+            optionflags=doctest.NORMALIZE_WHITESPACE |
+            doctest.ELLIPSIS |
+            doctest.IGNORE_EXCEPTION_DETAIL),
```

Comment:
```suggestion
            optionflags=(doctest.NORMALIZE_WHITESPACE 
                         | doctest.ELLIPSIS 
                         |doctest.IGNORE_EXCEPTION_DETAIL)
       ),
```

Since this is being changed anyway...Multi-line expressions with wrapping are [supposed to have the operators begin the line](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator), not end it.

The extra parentheses are for added clarity. I know that the grouping is technically implied because of the context, but "explicit is better than implicit" applies here?

(Don't trust the suggestion as-is, it's hard to indent in the GitHub comment editor.)

Reply:
I am going to fix the other `tests.py` files as well with the suggested change.